Content.Server/Mech/Systems/MechSystem.cs

```diff
@@ -454,4 +446,19 @@ private void OnExpose(EntityUid uid, MechPilotComponent component, ref AtmosExpo
         args.Handled = true;
     }
     #endregion
+
+    private sealed class RemoveBatteryEvent : EntityEventArgs
+    {
+
+    }
+
+    private sealed class MechEntryEvent : EntityEventArgs
+    {
+
+    }
+
+    private sealed class MechExitEvent : EntityEventArgs
+    {
+
+    }
```

Comment:
Please add the original comments back in.

Reply:
Fixed in https://github.com/space-wizards/space-station-14/pull/13225/commits/01f3c93afeb160420735aa628babce9e90b52fd9